python/lsst/pipe/tasks/assembleCoadd.py

```diff
@@ -619,7 +619,11 @@ def assembleMetadata(self, coaddExposure, tempExpRefList, weightList):
             self.inputRecorder.addVisitToCoadd(coaddInputs, tempExp, weight)
         coaddInputs.visits.sort()
         if self.config.doPsfMatch:
-            psf = self.config.modelPsf.apply()
+            # set PSF to modelPsf with maximum dimensions
```

Comment:
How much do you need here? Would it be sufficient to add explanation on why maximum dimensions in the commit message or would an inline message like the following help? 

            # modelPsf's BBox was dynamically defined by ModelPsfMatchTask
            # to be the square bounding the largest Warped Psf BBox